meta-balena-common/recipes-connectivity/networkmanager/resin-files/bluetooth.conf.systemd

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Wants=resin-net-config.service bind-var-lib-NetworkManager.service chronyd.service
+After=resin-net-config.service bind-var-lib-NetworkManager.service chronyd.service
```

Comment:
Struggling to understand why we need these 3 services before bluetooth?

Reply:
Fixed. It was a WIP artifact.